Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -138,9 +138,173 @@ public class PushDeploymentController : ApiController
             }
         }
 
-        private string GetZipURLFromARMJSON(JObject requestObject)
+        [HttpPost]
+        [HttpPut]
+        public async Task<HttpResponseMessage> OnePushDeploy(
+            [FromUri] string type = null,
+            [FromUri] bool async = false,
+            [FromUri] string path = null,
+            [FromUri] bool clean = false,
+            [FromUri] string message = DefaultMessage)
+        {
+            using (_tracer.Step("OnePushDeploy"))
+            {
+                var response = Request.CreateResponse();
+
+                var website_framework = System.Environment.GetEnvironmentVariable(Constants.Framework).ToLower();
```

Comment:
camelCasing: websiteFramework. Also, would be good to move this to a helper method in EnvironmentHelpers.cs. EnvironmentHelper::GetApplicationStack()